### DISCUSSION TITLE
summarytools::dfSummary in plumber API?

### DISCUSSION THREAD

**Discussion by skolenik** [dcomtois/summarytools#167] [General]
Should I reasonably expect that `dfSummary.print()` is good enough to be plugged into `plumber` as is? As of v 1.0.1, it does not work out of the box. I have posted on [RStudio Community](https://community.rstudio.com/t/use-summarytools-dfsummary-output-in-plumber/141514?u=stask) so I won't be replicating my reported issue and my reprex here on GitHub.

**Comment by dcomtois** [comment]
Sorry I left this unanswered... I have never used plumber... Did you ever get it to work?
